Tests/RoslynPluginGeneratorTests/NuGetMachineWideSettingsTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="NuGetMachineWideSettingsTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NuGet;
+using SonarQube.Plugins.Test.Common;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace SonarQube.Plugins.Roslyn.RoslynPluginGeneratorTests
+{
+    [TestClass]
+    public class NuGetMachineWideSettingsTests
```

Comment:
Tests that our new IMachineWideSettings implementation searches the expected directories.